roles/servicetelemetry/tasks/component_grafana.yml

```diff
@@ -16,17 +16,29 @@
 
 # Lookup existing datasources
 - block:
-  - name: Set default Prometheus datasource
+
+  - block:
+    - name: Retrieve elastic search secret
```

Comment:
I think this will fail when you don't have ElasticSearch deployed since the Secret won't exist?

Reply:
Disregard, I see the `when` now.